maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/mapper/MeansAssessmentMapper.java

```diff
@@ -173,7 +173,7 @@ private ApiIncomeEvidence mapToExtraApiIncomeEvidence(ExtraEvidenceDTO evidenceI
         return new ApiIncomeEvidence()
                 .withId(NumberUtils.toInteger(evidenceItem.getId()))
                 .withDateReceived(DateUtil.toLocalDateTime(evidenceItem.getDateReceived()))
-                .withApiEvidenceType(new ApiEvidenceType(evidenceType.getName(), evidenceType.getDescription()))
+                .withIncomeEvidence(evidenceItem.getEvidenceTypeDTO().getEvidence())
```

Comment:
Also separately, `evidenceType` defined on line 169 no longer looks to be being used.